<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(41, 44, 65);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1061</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
background-color:rgb(41, 44, 65);
}
QLabel{
color: white;
}
QPushButton {
background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-folder-open.png);
	background-position: left center;
padding: 5px;



    background-repeat: no-repeat;

	border: 1px solid transparent;
color:rgb(255, 255, 255);

border-radius:5px;
	border-left:5px solid rgba(61, 80, 235, 255);



	background-color:#3d50eb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 80, 235, 255), stop:1 rgba(118, 61, 235, 255));

}
QPushButton:hover {
color:rgb(255, 255, 255);
		background-color: rgba(61, 80, 235, 255);
}
QPushButton:pressed {	
	border: 2px solid rgb(55, 74, 214) ;
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>employee</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>921</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NOM">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PRENOM">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_DEP">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SAL">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SALAIRE:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ADRESSE:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>270</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-folder-open.png);
	background-position: left center;



    background-repeat: no-repeat;

	border: 1px solid transparent;
color:rgb(255, 255, 255);

border-radius:5px;
	border-left:5px solid rgba(61, 80, 235, 255);



	background-color:#3d50eb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 80, 235, 255), stop:1 rgba(118, 61, 235, 255));

}
QPushButton:hover {
color:rgb(255, 255, 255);
		background-color: rgba(61, 80, 235, 255);
}
QPushButton:pressed {	
	border: 2px solid rgb(55, 74, 214) ;
}</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>modiffier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-folder-open.png);
	background-position: left center;



    background-repeat: no-repeat;

	border: 1px solid transparent;
color:rgb(255, 255, 255);

border-radius:5px;
	border-left:5px solid rgba(61, 80, 235, 255);



	background-color:#3d50eb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 80, 235, 255), stop:1 rgba(118, 61, 235, 255));

}
QPushButton:hover {
color:rgb(255, 255, 255);
		background-color: rgba(61, 80, 235, 255);
}
QPushButton:pressed {	
	border: 2px solid rgb(55, 74, 214) ;
}</string>
        </property>
        <property name="text">
         <string>modiffier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NOM_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PRENOM_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_DEP_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SAL_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>250</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>160</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DEPARTEMENT:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SALAIRE:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-folder-open.png);
	background-position: left center;



    background-repeat: no-repeat;

	border: 1px solid transparent;
color:rgb(255, 255, 255);

border-radius:5px;
	border-left:5px solid rgba(61, 80, 235, 255);



	background-color:#3d50eb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 80, 235, 255), stop:1 rgba(118, 61, 235, 255));

}
QPushButton:hover {
color:rgb(255, 255, 255);
		background-color: rgba(61, 80, 235, 255);
}
QPushButton:pressed {	
	border: 2px solid rgb(55, 74, 214) ;
}</string>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>21</y>
          <width>331</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#25283b;
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_trier">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>90</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {	
border:none;
	background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-x.png);
	background-position: left center;
padding-left:5px;
    background-repeat: no-repeat;
background-color:rgb(232, 45, 76);
	border-radius: 5px;
border:1px solid transparent;
color:rgb(255, 255, 255);
	text-align: center;
border-left:3px solid rgba(232, 45, 113, 255);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(232, 45, 113, 255), stop:1 rgba(232, 45, 76, 255));
	
}
QPushButton:hover {
	background-color: rgba(232, 45, 76, 255);

}
QPushButton:pressed {
border:2px solid rgb(179, 34, 58);


}</string>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_desc_historique">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>60</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>descendant</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_asc_historique">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ascendant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idstock">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
border-style: solid;
border-color: white;
color: white;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="L_search">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>150</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {	
border:none;
background-color:#0dab9c;
	background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-description.png);
	background-position: left center;

    background-repeat: no-repeat;
border:1px solid transparent;
border-radius:5px;
	border-left:3px solid rgba(53, 171, 121, 255);

color:rgb(255, 255, 255);
	text-align: center;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(53, 171, 121, 255), stop:1 rgba(0, 171, 156, 255));
	
}
QPushButton:hover {
	background-color:rgba(53, 171, 121, 255);
	
}
QPushButton:pressed {
border:2px solid rgb(0, 132, 119);


}</string>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_ID">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>110</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_salaire">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>70</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>SALAIRE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_nom">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>30</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-image: url(C:/Users/HP/Documents/Health-application/icons/16x16/cil-folder-open.png);
	background-position: left center;



    background-repeat: no-repeat;

	border: 1px solid transparent;
color:rgb(255, 255, 255);

border-radius:5px;
	border-left:5px solid rgba(61, 80, 235, 255);



	background-color:#3d50eb;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 80, 235, 255), stop:1 rgba(118, 61, 235, 255));

}
QPushButton:hover {
color:rgb(255, 255, 255);
		background-color: rgba(61, 80, 235, 255);
}
QPushButton:pressed {	
	border: 2px solid rgb(55, 74, 214) ;
}</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>services</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>591</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TYPE:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prix_u">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>150</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>AJOUTER</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>modiffier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_modiffier">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>MODIFFIER</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TYPE:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_type_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TYPE:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mt">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>531</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: black;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_afficher">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>320</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_pdf">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>320</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
